powershell/plugins/create-commands-v2.ts

```diff
@@ -446,16 +445,42 @@ export /* @internal */ class Inferrer {
     //state.message({ Channel: Channel.Debug, Text: `${variant.verb}-${variant.subject} //  ${operation.operationId} => ${JSON.stringify(variant)} taking ${requiredParameters.joinWith(each => each.name)}; ${constantParameters} ; ${bodyPropertyNames} ${polymorphicBodies ? `; Polymorphic bodies: ${polymorphicBodies} ` : ''}` });
     await this.addVariant(pascalCase([variant.action, vname]), body, bodyParameterName, [...constants, ...requiredParameters], operation, variant, state);
 
-    const [pathParams, otherParams] = values(requiredParameters).bifurcate(each => each?.protocol?.http?.in === ParameterLocation.Path);
-    const dvi = await state.getValue('disable-via-identity', false);
-
-    if (!dvi && length(pathParams) > 0 && variant.action.toLowerCase() != 'list') {
-      // we have an operation that has path parameters, a good canididate for piping for identity.
-      await this.addVariant(pascalCase([variant.action, vname, 'via-identity']), body, bodyParameterName, [...constants, ...otherParams], operation, variant, state);
+    if (await state.getValue('disable-via-identity', false)) {
+      return;
     }
 
+    const disableGetEnableList = await this.state.getValue('ps-pipeline-input-disable-getByIteself-and-enable-listByParent', false);
```

Comment:
Suggest to change the configuration to something like `list-by-parent-pipeline` and default to `true`

Reply:
updated name to ''enable-parent-pipeline-input-for-list'' with default value true